orchestration/dagster_orchestration/hca_orchestration/solids/validate_egress.py

```diff
@@ -66,16 +67,19 @@ def base_notify_slack_of_egress_validation_results(
 ) -> str:
     gcp_env = context.solid_config["gcp_env"]
     dataset_name = context.solid_config["dataset_name"]
+    argo_workflow_id = context.solid_config["argo_workflow_id"]
 
     if validation_results.has_problems():
         message_lines = [
             f"Problems identified in post-validation for HCA {gcp_env} dataset {dataset_name}:",
+            f"Triggering Argo workflow ID: {argo_workflow_id}",
             "Duplicate lines found: " + str(validation_results.duplicates),
             "Null file references found: " + str(validation_results.null_file_refs),
             "Dangling project references found: " + str(validation_results.dangling_project_refs)
         ]
     else:
-        message_lines = [f"HCA {gcp_env} dataset {dataset_name} has passed post-validation."]
+        message_lines = [
+            f"HCA {gcp_env} dataset {dataset_name} workflow ID {argo_workflow_id} has passed post-validation."]
```

Comment:
I think we should follow the format from the problems branch and separate this into multiple lines, i.e., 

```suggestion
            f"HCA {gcp_env} dataset {dataset_name} has passed post-validation.",
            f"Argo Workflow ID: {argo_workflow_id}"
            ]
```

Reply:
agreed - this helps communicate that the workflow ID isn't coupled to the dataset, as well.